BOJ/2606.바이러스/6047198844.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <vector>
+#include <queue>
+
+using namespace std;
+
+bool check[101];
+vector <int> vt[101];
+
+int bfs(int computer_num) {
+	int res = 0;
+	memset(check, false, sizeof(bool) * 101);
```

Comment:
```suggestion
	memset(check, false, sizeof(check));
```
`sizeof` 함수의 인자로 배열을 전달하면 배열 전체 크기를 얻을 수 있습니다. 크기를 직접 명시하면 실수할 가능성이 있으니 이 방법을 권장드립니다.

Reply:
감사합니다. 혹시 이중배열의 경우도 배열 전체 크기를 얻을수있나요??

Reply:
@6047198844 배열은 메모리상에 연속으로 위치하기때문에 2차, 3차, ... N차 배열도 가능합니다!

Reply:
항상 답변 감사드립니다. 유용하게 쓸것같아요!!!